cmd/download.go

```diff
@@ -13,6 +13,11 @@ import (
 	"github.com/urfave/cli"
 )
 
+const (
+	submission_url_prefix        = "exercism.io/submissions/"
+	submission_url_prefix_w_http = "http://" + submission_url_prefix
```

Comment:
Would it be possible to use a precompiled regex here that optionally includes http:// (and, potentially, https://)?